src/styles/clock.scss

```
@@ -0,0 +1,61 @@
+.clock {
+
+  &__stopwatch {
+    @include square-size(600px);
+    border-radius: 50%;
+    border: 3px solid $main-theme-color;
+    position: relative;
+  }
+
+  &__btn:hover {
+    cursor: pointer;
+  }
+
+  &__btn:checked ~ * {
+    animation-play-state: running;
+  }
+
+  &__mainHand {
+    width: 48%;
+    height: 2%;
+    background-color: $main-theme-color;
+    
+    @extend %clock-hand-start-position;
+    @include arrow-animation(3600);
+  }
+  
+  &__center {
+    @include square_size(5%);
+    position: absolute;
+    left: 50%;
```

```suggestion
    left: 50%;
```
50% can be moved to variables